ui/app/adapters/watchable.js

```diff
@@ -98,6 +98,8 @@ export default class Watchable extends ApplicationAdapter {
       additionalParams,
       query
     );
+    // If your query is effectively a findAll under the hood, treat non-returned entities as deleted
+    const isPseudoFindAll = Object.values(params).every((v) => v === '*');
```

Comment:
`question`: The callback function in the `every` method assumes that `*` wildcard is applicable... However, I'm wondering the `types` of this params object specifically `queryString.parse(params)`.

```
  queryStringParams // based on URL in the browser which will create an object like {token: 'bada55cafe'}
```

Wouldn't we need a recursive operation here? Also note that if any query parameter that is not `namespace` is added then we can exit early knowing that it is not a `psuedoFindAll` operation.